src/manage_nsfs/manage_nsfs_cli_utils.js

```diff
@@ -96,13 +96,17 @@ function get_boolean_or_string_value(value) {
  * lead to unexpected output (usually array of NaN)
  * 1. if the value is a number return array with this number (3 => [3])
  * 2. if the value is a string return an array of numbers ('0,212,111' => [0,212,111])
- * 3. for all other types (including object and undefined) return the value itself
+ * 3. if value is an empty string (""). unset the value
```

Comment:
I think creating 2 sub indexes will be more clear:
```suggestion
* 2. if the value is a string:
 *   2.1 if value is an empty string (""). unset the value
 *   2.2 else return an array of numbers ('0,212,111' => [0,212,111])
```